src/test/java/com/iota/iri/benchmarks/crypto/CurlBenchmark.java

```diff
@@ -0,0 +1,14 @@
+package com.iota.iri.benchmarks.crypto;
+
+import com.iota.iri.crypto.CurlTest;
+import org.openjdk.jmh.annotations.Benchmark;
+
+public class CurlBenchmark {
+  private static CurlTest curlTest = new CurlTest();
+
+  @Benchmark
+  public void pairHashWorks() {
+    curlTest.pairHashWorks();
```

Comment:
In most cases, especially when it comes to production code, I would commend you for reusing code!!! :-)

Not here...
The reason is that testing code should have more of a "script-like" nature. Meaning that each test is a stand-alone unit that shouldn't change because of another test.

Imagine a dev adding a test case to `curlTest.pairHashWorks()`.
Then after a while we run the benchmark and see that performance was degraded.
So even though I am a big fan of DRY, here I make an exception and I will ask you to copy-paste